fdbserver/ClusterController.actor.cpp

```diff
@@ -337,19 +337,56 @@ class ClusterControllerData {
 
 		logServerSet = Reference<LocalitySet>(new LocalityMap<WorkerDetails>());
 		logServerMap = (LocalityMap<WorkerDetails>*)logServerSet.getPtr();
-		for (auto& it : id_worker) {
-			if (std::find(exclusionWorkerIds.begin(), exclusionWorkerIds.end(), it.second.details.interf.id()) ==
+
```

Comment:
can you also add a comment on `getWorkersForTlogs `

Reply:
Done.